api/models/vpp/interfaces/interface.proto

```diff
@@ -144,3 +146,30 @@ message VmxNet3Link {
     uint32 rxq_size = 3;    /* receive queue size (default is 1024) */
     uint32 txq_size = 4;    /* transmit queue size (default is 1024) */
 }
+
+message BondLink {
+    uint32 id = 1;
+    string mac = 2;
```

Comment:
Is this MAC address additional to phys_address from the common interface definition?

Reply:
Good question. The answer is no - it's the same mac address. Bond link has its own fields to set mac address, but it seems it does the same as the API call used for every other interface since only one can be set.